starter_code/app.js

```diff
@@ -8,11 +8,36 @@ const punkAPI = new PunkAPIWrapper();
 
 app.set('view engine', 'hbs');
 app.set('views', __dirname + '/views');
-app.use(express.static(path.join(__dirname, 'public')));
+app.use(express.static(__dirname + "/public"));
+hbs.registerPartials(__dirname + '/views/partials')
 
 app.get('/', (req, res, next) => {
   res.render('index');
 });
 
+app.get('/beers', (req, res, next) => {
+  punkAPI.getBeers()
+  .then(beers => {
+    res.locals.beersList = beers
+    res.render('beers.hbs');
+  })
+  .catch(error => {
+    console.log(error)
+  })
+
+
+});
+
+// app.get('/random-beer', (req, res, next) => {
+//   // punkAPI.randomBeer()
```

Comment:
Actually the good method is getRandom() !
You should have done this one, you were close ;)